compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -403,6 +411,31 @@ fn link_rlib<'a>(
         ab.add_file(&trailing_metadata);
     }
 
+    // add all native rlib dependencies
+    // archieves added to the end of .rlib archieve
```

Comment:
Need to explain why they are added to the end and not to other arbitrary position.

Reply:
Also "archieve(s)" -> "archive(s)".